src/compiler/scanner.rs

```diff
@@ -69,6 +70,27 @@ pub fn token_to_string(t: SyntaxKind) -> Option<&'static String> {
     token_strings.get(&t)
 }
 
+fn is_line_break(ch: char) -> bool {
```

Comment:
Some of these were in the wrong place as `Scanner` methods instead of top-level function helpers